SR2024/2023-12-13-challenge-reminder.md

```diff
@@ -0,0 +1,18 @@
+---
+to: Student Robotics 2024
+subject: Student Robotics Challenge submissions
+---
+
+Hi $TODO_CONTACT_GIVEN_NAME,
+
+We hope you had an excellent new year and holiday break.
+
+Congratulations to the 15 teams that completed one of the challenges in time for the first submission deadline in December. We've been really impressed with the challenge submissions this year, it's been great to see the creativity of so many teams so early on in the competition.
+
+If you aren't aware of the challenges, you can find out more about them in the [rulebook](https://studentrobotics.org/docs/resources/2024/rulebook.html#challenges)
+
+The next [challenge deadline](https://studentrobotics.org/events/sr2024/second-challenge-submission-deadline/) is 20th January 2024, so remember time will be short when your team gets back from Christmas break. There are another 5 league points up for grabs for anyone who manages to submit a challenge for this deadline.
+
+Your team can re-submit if their attempts aren't judged to have passed the challenge, so it's worth having a go in plenty of time before the deadline. If you have any questions, just ask us via e-mail or Discord.
+
+Thanks, $TODO_VOLUNTEER_NAME
```

Comment:
Just a though -- I think we'd normally sign off with like "-- The Competition Team" for general mailings.